### PR TITLE
Fixing reports on installers

### PR DESCRIPTION
**Pull request by simmonsj330**
# Description
Reports were crashing and failing on the installer. Now they're not after adding a hook and hidden imports / added_files to terminal.spec.

Note. Incorporating riskfolio and astropy into the bundle may add additional binaries to sign for the intel mac build automation. They might be automatically detected and signed during the automation process.

Forecast report working.
Screen Shot 2022-11-28 at 11 10 19 AM

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/reports/reports_controller.py [hunk 1]
```diff
@@ -145,9 +144,6 @@ def call_crypto(self, other_args: List[str]):
 
     @log_start_end(log=logger)
     def call_forecast(self, other_args: List[str]):
-        if is_packaged_application():
```
Comment (montezdesousa) [inline openbb_terminal/reports/reports_controller.py:148]: no need to have this check anymore?

**Review by montezdesousa — APPROVED** [review]

File: openbb_terminal/reports/reports_controller.py (hunk 1 above)
Comment (Chavithra) [reply openbb_terminal/reports/reports_controller.py:148]: it's disabling forecast

Comment (simmonsj330) [reply openbb_terminal/reports/reports_controller.py:148]: Negatory. We had to add that in to disable the forecast report on rc1

Comment (Chavithra) [reply openbb_terminal/reports/reports_controller.py:148]: Forecast working right now?

Comment (simmonsj330) [reply openbb_terminal/reports/reports_controller.py:148]: @Chavithra Yessirrrr. All reports are now thankfully

**Pull request merged**